meilisearch-core/src/store/mod.rs

```diff
@@ -363,10 +363,10 @@ impl Index {
         QueryBuilder::new(self)
     }
 
-    pub fn query_builder_with_criteria<'c, 'f, 'd, 'fa, 'i>(
+    pub fn query_builder_with_criteria<'c, 'f, 'd, 'fa, 'i, 'q>(
```

Comment:
Is this new `'q` lifetime necessary?

Reply:
And is it normal that we keep the `'fa` one?

Reply:
that's right, forgot to remove them here. Funny rustc does not warn about unused lifetimes...